apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -35,6 +41,12 @@ int tc_kernel_main(int argc, char *argv[])
 		return ERROR;
 	}
 
+	g_tc_fd = open(TASK_TESTCASE_DRVPATH, O_RDWR);
```

Comment:
Is there any change to write? Why O_RDWR?

Reply:
yes... so i changed it to O_WRONLY.